team-validator.js

```diff
@@ -215,6 +217,16 @@ class Validator {
 				return [`"${set.ability}" is an invalid ability.`];
 			}
 		}
+		nature = tools.getNature(set.nature);
+		if (nature.id && !nature.exists) {
```

Comment:
At which point this can simply be:

    if (!tools.getNature(set.nature).exists) {